mkosi/resources/mkosi-obs/mkosi.postoutput

```diff
@@ -46,10 +46,17 @@ for f in "${KERNELS[@]}"; do
     pesign --force -n sql:"$nss_db" -i "${OUTPUTDIR}/${f}" -E "hashes/kernels/$f"
 done
 
+repart_dir="$(jq -r .RepartDirectories[-1] < "$MKOSI_CONFIG")"
```

Comment:
`jq` does take file argument, so the input redirection shouldn't be necessary or is it?
```suggestion
repart_dir="$(jq -r '.RepartDirectories[-1]' "$MKOSI_CONFIG")"
```